config/default.schema.json

```diff
@@ -245,7 +252,33 @@
 		"S3_KEY": {
 			"type": "string",
 			"description": "Synchronous encryption key used to secure storage provider secrets. Only used with FEATURE_MULTIPLE_S3_PROVIDERS_ENABLED=true",
-			"pattern": ".{16}.*"
+			"pattern": ".{16}.*",
+			"default": "abcdefabcdefabcdefa"
+		},
+		"STORAGE_SIGNED_URL_EXPIRE": {
+			"type": "integer",
+			"default": 60,
+			"description": "Expiration time in seconds for signedUrl used by antivirus"
+		},
+		"ENABLE_FILE_SECURITY_CHECK": {
```

Comment:
add an if..then which makes the properties required, if ENABLE_FILE_SECURITY_CHECK (Which should have the name FEATURE_..._ENABLED) is true that uri/username/pw are required. This can be done by adding a definition to allOf

Reply:
I kept the name ENABLE_FILE_SECURITY_CHECK instead of FEATURE_ .... because this env var is already used and set in all the instances....